src/tests/ftest/util/command_utils.py

```diff
@@ -894,6 +893,9 @@ def __init__(self, namespace, command, path="", yaml_cfg=None, timeout=10, run_u
         # Default owner of the certificate files
         self.certificate_owner = getuser()
 
+        # Support disabling verifying the socket directory (runtime_dir) for tests
+        self.verify_socket_dir = True
```

Comment:
Looks like this is always True?

Reply:
Normally, yes, unless `SubprocessManager.verify_socket_dir = False` is called, like in `test_daos_agent_config_basic`

Reply:
I missed that this was just moved from one class to another